easybuild/easyblocks/b/basix.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+from easybuild.easyblocks.generic.cmakepythonpackage import CMakePythonPackage
+from easybuild.tools.run import run_cmd
+from easybuild.tools.build_log import EasyBuildError
+
+
+class EB_Basix(CMakePythonPackage):
+    """Custom easyblock for Basix"""
+    def build_step(self):
+        """Build step need not be overridden"""
+        super(EB_Basix, self).build_step()
+
+        # Building 'lib64/libbasix.so'
+        self.cfg['build_type'] = 'Release'
+        self.cfg['srcdir'] = 'cpp'
+        self.cfg['build_cmd'] = 'cmake --build %s/easybuild_obj' % self.builddir
+
+    def install_step(self):
+        """
+        Purpose:
+          1. Suppress the unwanted '--no-dep --ignore-cache' options of
+             the 'cmake --install <path>' command
+             Install the libbasix.so
+          2. Install the Python interface
+          3. Copy out lib the folder
+        """
+        self.cfg['installopts'] = ''
+        self.log.info('Resetting "installopts" in basix.py')
+
+        # step 1: Installing the C++ shared lib
```

Comment:
You are not installing anything here, are you? I'd expect a `super().install_step()` call at this point.

Why exactly do you need to clear `installopts`?